pandas/tests/plotting/test_frame.py

```diff
@@ -854,7 +847,7 @@ def test_area_lim(self):
     @pytest.mark.slow
     def test_bar_colors(self):
         import matplotlib.pyplot as plt
-        default_colors = self._maybe_unpack_cycler(plt.rcParams)
+        default_colors = [v['color'] for v in plt.rcParams['axes.prop_cycle']]
```

Comment:
Any reason you decided to replace the method with a list comp? The couple calls would probably be more readable if left alone